core/app/models/workarea/segment/rules/traffic_referrer.rb

```diff
@@ -13,11 +13,11 @@ def qualifies?(visit)
         end
 
         def medium_match?(referrer)
-          medium.to_s.strip.casecmp?(referrer.medium)
+          medium.to_s.present? && medium.to_s.strip.casecmp?(referrer.medium.to_s)
```

Comment:
```suggestion
          medium.to_s.strip.casecmp?(referrer.medium.to_s)
```

Reply:
That extra `to_s` and presence check doesn't get us anything.

Reply:
as discussed in standup, this preserves the logic because `.casecmp` changes behavior in 2.5 to return `false` when both sides are blank, whereas if both sides are blank in 2.4 and below, it will return `true`.